wire-ios-data-model/Source/Model/Conversation/ZMConversation+DisplayName.swift

```diff
@@ -33,7 +29,7 @@ public extension ZMConversation {
     var displayName: String {
         let result = meaningfulDisplayName
         switch conversationType {
-        case .oneOnOne, .connection: return result ?? ZMConversation.emptyConversationEllipsis
+        case .oneOnOne, .connection: return result ?? ""
```

Comment:
Wouldn't it be better to make `displayName` optional and return nil when we don't have one?